pylutron/__init__.py

```diff
@@ -916,7 +916,7 @@ class Event(LutronEvent):
 
     STATE_CHANGED: The button has been pressed.
         Params:
-          state: The boolean value of the new LED state.
+          state: The value of the new LED state. 0 = off, 1= on, 2= 1 flash/sec, 3= 10 flash/sec
```

Comment:
line too long. I'd put the possible values on the next line (or two). Also, please correct your spacing around the =

Also, we'd need to figure out how to represent the various states of the LED to let the user control them. Just passing the arbitrary Lutron defined integers is pretty useless.

Reply:
Agreed, This part has to be completed